packages/squiggle-lang/src/library/FrInput.ts

```diff
@@ -0,0 +1,52 @@
+import { FnInput } from "../reducer/lambda/FnInput.js";
+import { FrType } from "./FrType.js";
+
+type Props<T> = {
+  name?: string;
+  type: FrType<T>;
+};
+
+export class FrInput<IsOptional extends boolean, Unpacked = unknown> {
```

Comment:
The name is annoying because we also had `frInput` which is "fr type for VInput". Because of this, I renamed `frInput` to `frFormInput`, and we probably should rename `VInput` to `VFormInput` too.

Reply:
Really looking forward to LLM systems that could do broader refactors for gradually improving things like these.